entrypoint.sh

```diff
@@ -15,53 +15,69 @@ SCRIPT_AFTER_MAKE_TEST="`pwd`/.github/ci-bionic/after_make_test.sh"
 
 cd $GITHUB_WORKSPACE
 
-echo ::group::Dependencies from binaries
+echo ::group::Install tools: apt
 apt update
 apt -y install wget lsb-release gnupg
 sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -cs) main" > /etc/apt/sources.list.d/gazebo-stable.list'
 apt-key adv --keyserver keyserver.ubuntu.com --recv-keys D2486D2DD83DB69272AFE98867170598AF249743
-apt-get update
+apt-get update 2>&1
```

Comment:
why do we need to redirect the apt-get update?

Reply:
That spits out "WARNING: apt does not have a stable CLI interface." to stderr, which leaks out of the `::group::`.